lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -221,6 +224,15 @@ namespace :foreman_tasks do
         tasks.order('started_at desc').all.each do |task|
```

Comment:
Before this change `tasks` was a `ForemanTasks::Task::ActiveRecord_Relation`, now it is an `Array` of `ForemanTasks::Task`, which does not implement `order`

```
NoMethodError: undefined method `order' for #<Array:0x000000000b043480>
lib/foreman_tasks/tasks/export_tasks.rake:224:in `generate_index'
lib/foreman_tasks/tasks/export_tasks.rake:284:in `block (4 levels) in <main>'
lib/foreman_tasks/tasks/export_tasks.rake:284:in `open'
lib/foreman_tasks/tasks/export_tasks.rake:284:in `block (3 levels) in <main>'
lib/foreman_tasks/tasks/export_tasks.rake:262:in `block (2 levels) in <main>'
```